core/templates/dev/head/components/SolutionEditorDirective.js

```diff
@@ -80,6 +80,7 @@ oppia.directive('solutionEditor', [
                       /* eslint-disable max-len */
                       explorationStatesService.getInteractionCustomizationArgsMemento(
                         EditorStateService.getActiveStateName()),
+                        true,
```

Comment:
Deindent by 2 and put this below the eslint-enable. This is a separate arg from the previous one.

Reply:
Done